Tests/scripts/update_id_set.py

```diff
@@ -396,6 +460,76 @@ def re_create_id_set():
     exec_time = time.time() - start_time
     print_color("Finished the creation of the id_set. Total time: {} seconds".format(exec_time), LOG_COLORS.GREEN)
 
+    duplicates = find_duplicates(new_ids_dict)
+    if duplicates:
```

Comment:
`duplicates = [[], [], [], []]` is a `true` expression but indicates that there are no duplicates.
```suggestion
    if any(duplicates):
```